config/.eslintrc.js

```diff
@@ -32,6 +32,10 @@ module.exports = {
     ],
     rules: {
         "@typescript-eslint/explicit-function-return-type": "off",
+        "@typescript-eslint/explicit-module-boundary-types": "off",
+        "@typescript-eslint/no-explicit-any": "off",
```

Comment:
I have mixed feelings about this one. On the one hand the rule can be annoying especially when dealing with complex type errors that you just want to go away. But having gone through several large scale refactorings lately, I've started to despise the `as any` escape hatch. It's definitely technical debt.

However, for an SDK targeting people who might not be experts in TypeScript's advanced typings, then this is probably OK. But I'm seriously considering enabling this rule in GXW itself.